### DISCUSSION TITLE
Whistleblower Identity Management

### DISCUSSION THREAD

**Discussion by techbriplast** [globaleaks/globaleaks-whistleblowing-software#3497] [General]
Hi , I would activate "Whistleblower Identity Management" but I can't view any check in context like https://github.com/globaleaks/GlobaLeaks/issues/185 says .
Thanks

**Comment by evilaliv3** [comment]
Thank you for your question.

To enable the identity management you need also to enable the Custodian user role (in advanced settings) and add to the system at least a user with the role Cuistodian

**Comment by techbriplast** [comment]
Hi evilaliv3, thanks for your reply.
yes, I had already done that : new custodian user (without administrative access to settings), logged in with it (to be sure) , created a new context but in advanced settings of this new context I can't see any Whistleblower Identity Management check .
Could it be related to site mode? I don't know if it will affect , I have no idea ... but I don't know where to look anymore .

thanks for your help

**Comment by suhuuraqaali** [comment]
Hi

**Comment by mauriziobulgarini** [comment]
Hi, in my experience you don't need any special configuration on the channel, you just need to add to your questionnaire the template-question "Do you want to provide your identification information?". You can find it using the button "Add question from template".

**Comment by cienne1** [comment]
Hi. I've already added to the system a user with the role Cuistodian, but I can't find any template related to identity management. Where exactely is it? Tks

**Comment by mauriziobulgarini** [comment]
<img width="1509" alt="Schermata 2023-08-07 alle 14 03 36" src="https://github.com/globaleaks/GlobaLeaks/assets/20863823/daa0eabb-6533-4af7-ab57-84b847227896">

**Reply by mauriziobulgarini** [reply]
Add the question from template in a questionnaire

**Comment by cienne1** [comment]
Grazie Maurizio. Passo all'italiano, se non è un problema. Siamo riusciti a configurare il template per i dati identificativi. Il sistema ha creato un "questionario aggiuntivo", in cui chiede al segnalante se vuole fornire i dati identificativi. Non prevede pero' la possibilità di scegliere si/no. Se non si compilano i dati la segnalazione non puo' essere inviata.

![image](https://github.com/globaleaks/GlobaLeaks/assets/141617580/405283ab-e44b-4c7d-a8c0-daea89e9e1bc)

**Reply by mauriziobulgarini** [reply]
Sembra un comportamento anomalo, la domanda dovrebbe uscire come nel mio screenshot.
Prova a verificare le impostazioni di lingua (se usi più di una lingua magari la domanda non è stata correttamente tradotta in italiano) e magari prova anche a inserire la domanda nel questionario principale (evita in pratica il questionario aggiuntivo che non mi pare comunque funzionale allo scopo)

![image](https://github.com/globaleaks/GlobaLeaks/assets/20863823/59c887a3-7668-4fff-bc77-83a2506b3092)

**Comment by cienne1** [comment]
Grazie mille. Il nostro problema credo sia a monte a questo punto, perchè non sappiamo come inserire la domanda nel questionario principale.

**Reply by mauriziobulgarini** [reply]
Mi viene in mente che magari state cercando di modificare il questionario Standard (quello denominato GLOBALEAKS). Ti consiglio di crearne uno nuovo oppure duplicarlo in modo da averne uno modificabile a tuo piacimento. A questo punto potrai aggiungere la domanda per il Custode. Ricordati poi di associare il nuovo questionario creato al canale di segnalazione.
Buon lavoro!

**Reply by cienne1** [reply]
RISOLTO. Grazie mille!
